rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -153,8 +154,7 @@ ok(rclcpp::Context::SharedPtr context = nullptr);
  * \param[in] context Check for initialization of this Context.
  * \return true if the context is initialized, and false otherwise
  */
-RCLCPP_PUBLIC
-[[deprecated("use the function ok() instead, which has the same usage.")]]
+RCLCPP_PUBLIC RCLCPP_DEPRECATED("use the function ok() instead, which has the same usage.")
```

Comment:
Note, I collapsed these into one line to satisfy uncrustify :man_shrugging:

Reply:
Looking at this again, I was wondering whether we had this situation in the past.  And it turns out we did; if you look at https://github.com/ros2/rclcpp/pull/852/files#diff-dccf0fec5a09dca9cbb3c47fd669be36L173 , it's the same thing.  So I'm guessing if you just reverse the order of the visibility macro and deprecated statement, clang will probably be OK with it.  Worth a try, at least.

Reply:
I'll give it a try.

Reply:
Changing the order works locally for me. I've updated this PR (7b466a6) and will re-trigger CI to be sure.